audio-worklet/src/bitcrusher.js

```diff
@@ -40,72 +24,70 @@ class Bitcrusher {
    * @param {Number} options.bitDepth the number of bits for each sample
    * @param {Number} options.reduction the degree of reduction
    */
-  constructor (context, options) {
-    if (context == null) throw "No AudioContext given";
-    if (options == null) options = {}; 
+  constructor(context, options) {
+    if (context == null) throw 'No AudioContext given';
+    if (options == null) options = {};
     this.context_ = context;
-    
-    //... Default to maximum bit depth and no reduction and let browser decide
-    //... buffersize unless specified
-    this.bitDepth = options.bitDepth || 24; 
+
+    // Default to maximum bit depth and no reduction and let browser decide
+    // buffersize unless specified.
+    this.bitDepth = options.bitDepth || 24;
     this.reduction = options.reduction || 1;
     let bufferSize = options.bufferSize || 0;
     let inputChannels = options.inputChannels || 1;
     let outputChannels = options.inputChannels || 1;
 
-    this.node_ = this.context_.createScriptProcessor(bufferSize, 
-      options.inputChannels, options.outputChannels);
-
-    this.node_.onaudioprocess = this.onaudioprocess_.bind(this); 
+    this.node_ = this.context_.createScriptProcessor(
```

Comment:
This is not AudioWorkletNode. I think this file should not be included in the PR until it's ready for the review.